src/main/java/seedu/simplykitchen/model/food/ExpiryDate.java

```diff
@@ -16,9 +16,8 @@
 public class ExpiryDate {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "The expiry date should be of the format DD-MM-YYYY or DD/MM/YYYY. ";
-    public static final String MESSAGE_SHORTENED_YEAR = "The year should be 4 digits long. ";
-    public static final String MESSAGE_INVALID_DATE = "The expiry date does not exist. ";
+            "The expiry date should be of the format DD-MM-YYYY or DD/MM/YYYY. "
+                    + "The year should be between 2020 and 2120, both inclusive.";
```

Comment:
Should we include an error message for invalid date? For example 29-2-2021 is the correct format and is between 2020 and 2120, but is not valid.

Reply:
I edited the message to include the valid date part.

The two date formatters used check for different types of formatting errors and also invalid date as well. Hence, the formatters can't differentiate between an invalid date or a date with incorrect formatting.

Reply:
Oh I see. Thanks!